lib/core-integration/src/Test/Integration/Scenario/API/Network.hs

```diff
@@ -0,0 +1,40 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Integration.Scenario.API.Network
+    ( spec
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.Api.Types
+    ( ApiNetworkInformation )
+import Cardano.Wallet.Primitive.Types
+    ( NtpStatus (..), ProtocolUpdates (..), SyncProgress (..) )
+import Test.Hspec
+    ( SpecWith, it )
+import Test.Integration.Framework.DSL
+    ( Context (..)
+    , Headers (..)
+    , Payload (..)
+    , eventually
+    , expectFieldEqual
+    , ntpStatus
+    , protocolUpdates
+    , request
+    , syncProgress
+    , verify
+    )
+
+spec :: forall t. SpecWith (Context t)
+spec = do
+    it "NETWORK - Can query network information" $ \ctx -> do
+        let endpoint = ("GET", "v2/network/information")
```

Comment:
A tests against `405` could be also added, like https://github.com/input-output-hk/cardano-wallet/blob/f0f721bdaff148cab01299334833a583b01a8119/lib/core-integration/src/Test/Integration/Scenario/API/Wallets.hs#L855

Reply:
will do in a separate PR (with the CLI)